.changeset/slimy-sloths-fry.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': patch
+---
+
+Fix the problem getViteConfig() duplicates some config
```

Comment:
```suggestion
Fixes a problem where using `getViteConfig()` resulted in an incorrect confirmation.
```

Reply:
@ematipico Thank you for your review!

I commited your suggestion: https://github.com/withastro/astro/pull/12312/commits/98b2e90169fecac386434f3e63eeacecbc0c4ff3